packages/design-system/src/theme.ts

```diff
@@ -53,8 +59,14 @@ export const theme: Theme = {
       800: '#150747',
       900: '#07021d',
     },
+    diamond: '#40e8ec',
+    platinum: '#81b6e3',
+    gold: '#d0a757',
+    silver: '#b0b0b0',
+    bronze: '#a97142',
     offwhite: '#F6F8F9',
     blue02: 'rgba(79, 105, 205, 0.2)',
+    dark60: 'rgba(0,0,0, 0.6)',
```

Comment:
we should come up with a more readable and uniform naming convention for translucent colors.
black60 sounds better to me than dark60. 
Maybe I should change blue02 to blue20 as well.

Reply:
Yea, want to consolidate the colours a lot more. e.g. purpleBoxDark should just be using purple.800, purple tag should be purple.400, etc. Will update blue02 to 20 though.